content/02.main-text.md

```diff
@@ -63,6 +63,13 @@ Edits made by distinct authors may be attributed to a single author.
 The GitHub Sync feature attributes all edits to the project owner.
 {#tbl:platforms}
 
+Existing platforms work well for editing text and are widely used for scholarly writing.
+However, they often lack features that are important for open collaborative writing, such as versatile version control and multiple permission levels.
+Table @tbl:platforms compares several existing platforms.
```

Comment:
This sentence seems redundant to me.  Can we cut it and add the table reference elsewhere in this paragraph in a parenthetical if you think we need it?